homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -361,6 +416,25 @@ async def async_setup_entry(
             EnvoyConsumptionEntity(coordinator, description)
             for description in CONSUMPTION_SENSORS
         )
+    # For each production phase reported add production entities
+    if envoy_data.system_production_phases:
+        for usephase in envoy_data.system_production_phases:
```

Comment:
Please separate words with _

Reply:
Done (need to get rid of this old habbit)